core/src/components/progress-bar/progress-bar.tsx

```diff
@@ -57,10 +57,35 @@ export class ProgressBar implements ComponentInterface {
    */
   @Prop({ reflect: true }) color?: Color;
 
+  /**
+   * Set to `"round"` for a progress-bar with rounded corners, or `"rectangular"`
+   * for a progress-bar without rounded corners.
```

Comment:
```suggestion
   * Set to `"round"` for a progress bar with rounded corners, or `"rectangular"`
   * for a progress bar without rounded corners.
```

This would be consistent with our documentation.